leshan-client-cf/src/main/java/org/eclipse/leshan/client/californium/LeshanClientBuilder.java

```diff
@@ -123,6 +123,16 @@ public LeshanClientBuilder setDtlsConfig(DtlsConnectorConfig.Builder config) {
         return this;
     }
 
+    public static NetworkConfig createDefaultNetworkConfig() {
+        NetworkConfig networkConfig = new NetworkConfig();
+        networkConfig.set(Keys.MID_TRACKER, "NULL");
+        networkConfig.set(Keys.MAX_ACTIVE_PEERS, 10);
+        networkConfig.set(Keys.MAX_MESSAGE_SIZE, 4096);
```

Comment:
MAX_RESOURCE_BODY_SIZE instead?
I don't know what is a "good" default value, so I would use the Californium default one.